spec/helpers/application_helper_spec.rb

```diff
@@ -364,4 +364,50 @@
       end
     end
   end
+
+  describe "#commenter_id_for_css_classes" do
+    context "when commenter is a user" do
+      let(:user) { create(:user) }
+      let(:comment) { create(:comment, pseud: user.default_pseud) }
+
+      it "returns commenter id css class name" do
+        expect(helper.commenter_id_for_css_classes(comment)).to eq("user-#{user.id}")
+      end
+    end
+    
+    context "when commenter is a visitor" do
+      let(:comment) { create(:comment, :by_guest) }
+
+      it "returns nil" do
+        expect(helper.commenter_id_for_css_classes(comment)).to eq(nil)
+      end
+    end
+
+    context "when commenter is creater of work inside anonymous collection" do
```

Comment:
Should be "creator." This could even go under `context "when commenter is a user"`, since the commenter here is indeed a user.